source/reference/verification.txt

```diff
@@ -0,0 +1,211 @@
+
+####################
+Verify Data Transfer
+####################
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Following a sync, verify the successful transfer of your data before you switch
+your application from using the source cluster to the destination cluster. You
+can verify your data transfer using document counts, hash comparison, document
+comparison, or the Migration Verifier.
+
+Use Cases
+=========
+
+You should verify your data after every sync. This is important in
+cases where you plan to move your application load from the source to the
+destination cluster.
+
+Verification methods include the comparison of
+:ref:`document counts <c2c-verify-doc-counts>`,
+comparison of :ref:`database hashes <c2c-verify-hash-comp>`,
+scripting :ref:`document comparisons <c2c-verify-doc-comp>`,
+and the :ref:`Migration Verifier <c2c-verify-migration-verifier>`.
+The specific method you use to verify your data depends on your application
+workload and the complexity of the data.
+
+Details
+=======
+
+.. _c2c-verify-doc-counts:
+
+Document Counts
+---------------
+
+The most basic method of verifying sync is by comparing the number of documents
+in each synced collection on the source cluster to the number on the
+destination cluster.
+
+Compare the results on each cluster:
+
+- If the document count is different,  not all changes synced to the
+  destination cluster.
+
+- If the document count is the same, the destination cluster has the same
+  number of documents. This doesn’t guarantee a successful sync of all changes
+  to those documents, except in cases where you have an insert-only workload.
```

Comment:
I want to make sure I'm understanding this sentence correctly:

> This doesn’t guarantee a successful sync of all changes
  to those documents, except in cases where you have an insert-only workload.

Is this saying that verifying that the document counts are the same does not ensure that modifications to the data (as in update operations) are accounted for in the sync? As in, the document counts might be the same but that doesn't necessarily mean that the correct data was synced?

Reply:
Reworked for clarity.